.applier/group_vars/seed-hosts.yml

```diff
@@ -60,7 +60,7 @@ openshift_cluster_content:
     template: "https://raw.githubusercontent.com/redhat-cop/containers-quickstarts/master/jenkins-masters/hygieia-plugin/.openshift/templates/jenkins-s2i.yml"
     namespace: "{{ ci_cd_namespace }}"
   - name: Deploy Jenkins Master
-    template: "https://raw.githubusercontent.com/redhat-cop/containers-quickstarts/master/jenkins-masters/hygieia-plugin/.openshift/templates/jenkins-ephemeral.yml"
+    template: "{{ inventory_dir }}/../.openshift/templates/jenkins-ephemeral.yml"
```

Comment:
I'm confused. Is this not what we're trying to avoid? this ephemeral jenkins template is the same in containers-quickstarts, no?

Reply:
Yep, that was a band-aid to get the thigns working. we can change it to a local reference